feature-libs/product-configurator/rulebased/components/group-title/configurator-group-title.component.ts

```diff
@@ -41,31 +50,67 @@ export class ConfiguratorGroupTitleComponent {
   iconTypes = ICON_TYPE;
 
   //TODO(CXSPA-1014): make ConfiguratorExpertModeService a required dependency
```

Comment:
Please enhance comment with the other new dependencies

Reply:
Done!